tests/Microsoft.Identity.Test.Unit/ExceptionTests/MsalExceptionTests.cs

```diff
@@ -432,6 +433,24 @@ public void ExceptionsPropertiesHavePublicSetters()
             AssertPropertyHasPublicGetAndSet(typeof(MsalServiceException), "CorrelationId");
         }
 
+        [TestMethod]
+        public async Task CorrelationIdInExceptions()
+        {
+            var ex = await AssertException.TaskThrowsAsync<MsalUiRequiredException>(async () =>

```

Comment:
This is fine, but MsalUIRequiredException already had this correlation ID. 

How about adding (or changing) a test that results in a real MsalClientException? Maybe this one - https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/Internal/JsonWebToken.cs#L158

Reply:
Yeah, I was looking for a way to trigger the client exception. this will work. thanks